src/context/wishListContext.jsx

```diff
@@ -0,0 +1,19 @@
+import { createContext, useContext, useReducer } from 'react';
+import { WishListReducer } from '../../src/reducers/wishlistReducer';
+
+const wishListContext = createContext(null);
+
+function WishListProvider( { children } ){
+
+    const [ wishList, setWishList ] = useReducer( WishListReducer, [])

```

Comment:
suggestion - here `setWishlist` is giving an idea that it's a `useState` but for `useReducer`, as a convention, it is named as `dispatch` or `somethingDispatch` .